### ISSUE TITLE
Original/2nd/F/X song support

### ISSUE THREAD

**Issue by Epicpkmn11** [Hydr8gon/Project-DS#2]
I tried some songs from F 2nd (Vita) to see if they would work, and while the music preview worked fine trying to play the song ended instantly.

If you think this would be good to add but don't want to yourself, I could try doing it. Since Open PD Script Editor supports them I assume it probably shouldn't be that hard to parse the DSCs, might be a bit tricky to draw held notes...

**Comment by Hydr8gon** [comment]
It would definitely be neat to have this eventually, but there are quite a few differences between the games that would need to be supported. Aside from the different note types, score and clear percent also work completely differently. I'm not sure what the best way to integrate this would be, but it's not a huge priority for me at the moment since most F songs are in Mega Mix/Future Tone and the X songs have been ported unofficially.
